components/support/versioning/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "versioning"
```

Comment:
This is specific to Firefox versioning so the title should probably reflect that.